x-pack/test/saved_object_api_integration/common/lib/test_data_loader.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
```

Comment:
@azasypkin I recreated the original testDataLoader in this directory for now since it requires a local `FtrProviderContext`; do you think moving it up to a parent directory makes sense?